### DISCUSSION TITLE
Relationships and common ancestors

### DISCUSSION THREAD

**Discussion by ghezibde** [vesta-webtrees-2-custom-modules/vesta_extended_relationships#96] [Ideas]
Hi Richard,
Something i'm wondering about each time i'm using Relationship trying to find all relationships via common ancestors is the number of ancestors for each found relationship and thus the total common ancestors for two indis in the tree.
Would it be feasible to compute and display that information in the layout ?
Let's say something like this:
![image](https://user-images.githubusercontent.com/75539399/214160049-0dca0236-0fce-4489-bcc0-755b08cf6989.png)
Regards,
André

**Comment by ric2016** [comment]
> the number of ancestors for each found relationship

This is just (the number of boxes -2), I'm not sure how useful that number would be.

The total number should omit duplicates, probably, and isn't really that meaningful either in my opinion (what about common ancestors 'further up' the tree?)

In what way do you consider these numbers to be significant?
